### PR TITLE
chore: Update dependabot.yml

### PR DESCRIPTION
**Pull request by gkarat**
This aims to simplify the dependabot config for Compliance and also make it consistent with other Insights apps' configs.

### CODE REVIEW

**Comment by AsToNlele** [comment]
bruh jenkins

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,146 +1,18 @@
...
+      - "dependencies"
+    open-pull-requests-limit: 5
+    commit-message:
+      prefix: "chore(deps):"
+    allow:
+      - dependency-name: "@redhat-cloud-services/frontend*"
+      - dependency-name: "@patternfly/*"
+        dependency-type: direct
+    ignore:
+      - dependency-name: "react-router-dom"
```
Comment (bastilian) [inline .github/dependabot.yml:17]: I'm not sure if it's worth adding this, given we'll upgrade soon anyways.

**Review by bastilian — APPROVED** [review]
Awesome! Thank you @gkarat!

**Pull request merged**